### PR TITLE
[WIP] maintainers-tools: Readme with local language support

### PR DESCRIPTION
**Pull request by victormmtorres**
### Milestone (Odoo version)
- 

### Module(s)
- 

### Fixes / new features
- Add language localization

cc @Tecnativa

### CODE REVIEW

File: tools/gen_addon_readme.template.es [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+{% endfor %}
+{% for _ in badges %}|badge{{ loop.index }}| {% endfor %}
+
+{{ fragments.get('DESCRIPTION', '') }}
+**Table of contents**
+
+.. contents::
+   :local:
+
+{{ fragment('INSTALL', 'Instalción') }}
```
Comment (elicoidal) [inline tools/gen_addon_readme.template.es]:
```suggestion
{{ fragment('INSTALL', 'Instalación') }}
```

File: tools/gen_addon_readme.template.es [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+{{- fragment('CONFIGURE', 'Configuración') }}
+{{- fragment('USAGE', 'Uso') }}
+{{- fragment('DEVELOP', 'Desarrollo') }}
+{{- fragment('ROADMAP', 'Incidencias conocidas / Hoja de ruta') -}}
+{{- fragment('HISTORY', 'Changelog') -}}
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/{{ org_name }}/{{ repo_name }}/issues>`_.
```
Comment (elicoidal) [inline tools/gen_addon_readme.template.es]: Why not translate all the content?

File: tools/gen_addon_readme.py [hunk 3]
```diff
@@ -282,8 +289,13 @@ def gen_one_addon_index(readme_filename):
               help="git commit changes to README.rst, if any.")
 @click.option('--gen-html/--no-gen-html', default=True,
               help="Generate index html file.")
+@click.option('--lang', default='',
+              help="Target language file code (default is English)"
+                   "List of supported codes: "
+                   "[(--lang=es for Español),]")
```
Comment (elicoidal) [inline tools/gen_addon_readme.py]:
```suggestion
                   "[(--lang=es for Spanish),]")
```

Comment (elicoidal) [reply tools/gen_addon_readme.py]: Somehow feels short: you should use the country+lang code.
What about MX,etc...

**Comment by victormmtorres** [comment]
Changes done @elicoidal

File: tools/gen_addon_readme.py [hunk 4]
```diff
@@ -282,8 +292,13 @@ def gen_one_addon_index(readme_filename):
               help="git commit changes to README.rst, if any.")
 @click.option('--gen-html/--no-gen-html', default=True,
               help="Generate index html file.")
+@click.option('--lang', default='',
+              help="Target language file code (default is en_En)"
+                   "List of supported codes: "
+                   "[(--lang=es for es_ES),]")
```
Comment (elicoidal) [inline tools/gen_addon_readme.py]: I am not sure that covers my point. How would you code MX Spanish?

Comment (victormmtorres) [reply tools/gen_addon_readme.py]: Thought in principle to have just one file per language to not do it more complicated but I understand your concern. So what about now with language + country codes?

Comment (elicoidal) [reply tools/gen_addon_readme.py]: Indeed that was my intention originally: use standard language+country code @victormmtorres

Comment (victormmtorres) [reply tools/gen_addon_readme.py]: Think it's done, could you update your review?

**Comment by pedrobaeza** [comment]
@sbidoul how do you think we can enable this by repositories for example for having the bot populating Spanish template in OCA/l10n-spain?

File: tools/gen_addon_readme.template.es_Es [hunk 5]
```diff
@@ -0,0 +1,102 @@
+﻿{%- macro fragment(name, title, sub='=') %}
```
Comment (sbidoul) [inline tools/gen_addon_readme.template.es_Es]: In the file name: es_Es => es_ES ?

**Comment by sbidoul** [comment]
Instead of passing a `--lang` option we could maybe take the language from the `readme` directory name. So if `readme/` is present, generate for the default english template. If `readme.es_ES/` is present, generate for es_ES, etc.

**Comment by pedrobaeza** [comment]
Good idea, @sbidoul 

@victormmtorres can you implement this?

**Comment by sbidoul** [comment]
Question: should allow for multiple readme's? (README.rst, README.es_ES.rst)

**Comment by pedrobaeza** [comment]
That's something I have think about, and the visibility in the apps store, in Odoo, in GitHub, but starting with, people is not going to translate too much the READMEs, unless we provide a Weblate like platform for that, so for now, I would go for this solution for l10n-xx repos, and then we can think about mechanism for what you say.

**Comment by rafaelbn** [comment]
@victormmtorres please check what @sbidoul said https://github.com/OCA/maintainer-tools/pull/414#issuecomment-502802644

**Comment by rafaelbn** [comment]
@pedrobaeza maybe other team member can finish this one

**Comment by pedrobaeza** [comment]
The approach should be another, so closing.

**Pull request closed without merging**